app/hypnos/src/main.c

```diff
@@ -52,7 +56,6 @@ void main_thread(void)
 		LOG_INF("Disabling BLE advertising...");
 		bt_adv_stop();
 		cts_sync_enable(false);
-		k_cpu_idle();
```

Comment:
This doesn't belong to this PR. Please do any power-related changes in a separate branch. I've already measured that k_cpu_idle() does makes a different in power consumption so I don't want to see it removed without motivation.